.pre-commit-config.yaml

```diff
@@ -0,0 +1,24 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
Fix indentations.

Reply:
Ah, this was generated pre-commit, but will align with other stuff

Reply:
fixed 4f3f37c